backend/sites/Dockerfile

```diff
@@ -18,10 +18,9 @@ ENV NODE_ENV=production
 WORKDIR /app
 COPY --from=build /app/node_modules ./node_modules
 COPY --from=build /app/dist ./dist
-
+COPY ./schema.graphql  ./schema.graphql
```

Comment:
This file should be generated at runtime. In fact I just merged a PR to remove it from source control since it's constantly showing as dirty.

Reply:
> This file should be generated at runtime. In fact I just merged a PR to remove it from source control since it's constantly showing as dirty.

Thanks a lot @bolny , you are a big saver.